packages/design-systems/src/components/composite/Datagrid/SearchFilter/FilterRow.tsx

```diff
@@ -152,20 +169,34 @@ export const FilterRow = <TData extends Record<string, unknown>>(
     return selectedColumnObject?.meta?.type || "text";
   }, [selectedColumnObject?.meta?.type]);
 
+  const renderValueInput: ValueInputType = useMemo(() => {
+    // render pattern
+    if (currentFilter.condition === "in") {
+      return "IN";
+    }
+    if (selectedColumnObject?.meta?.type === "enum") {
+      return "ENUM";
+    }
+    if (selectedColumnObject?.meta?.type === "boolean") {
+      return "BOOLEAN";
+    }
+    return "OTHER";
+  }, [currentFilter.condition, selectedColumnObject?.meta?.type]);
```

Comment:
排他であってfallthroughさせたいわけじゃないと思うのでelse-ifで分岐してください

```suggestion
  const renderValueInput: ValueInputType = useMemo(() => {
    // render pattern
    if (currentFilter.condition === "in") {
      return "IN";
    } else if (selectedColumnObject?.meta?.type === "enum") {
      return "ENUM";
    } else if (selectedColumnObject?.meta?.type === "boolean") {
      return "BOOLEAN";
    } else {
      return "OTHER";
    }
  }, [currentFilter.condition, selectedColumnObject?.meta?.type]);
```

Reply:
修正しました！
https://github.com/tailor-platform/frontend-packages/pull/236/commits/a13358242256a2db00bfff70bf877511d30b6e7d